os/Kconfig.debug

```diff
@@ -529,6 +529,46 @@ config DEBUG_TTRACE
 	---help---
 		Enable T-trace debug SYSLOG output (disabled by default)
 
+comment "System Debug Options"
+
+config DEBUG_SYSTEM
+	bool "System debug"
+	default n
+	depends on DEBUG_VERBOSE
```

Comment:
1. Why do you make a dependancy with DEBUG_VERBOSE? You use "dbg" and it is enabled when DEBUG_ERROR is enabled.
2. Let's use "select" instead of "depends on". "select" is better to use and DEBUG_VERBOSE doesn't have any dependancy so that we can use "select".